lib/ae/base/ae_template.py

```diff
@@ -0,0 +1,115 @@
+import ufe
+import maya.internal.common.ufe_ae.template as ufeAeTemplate
+
+from pxr import UsdGeom, UsdShade
+
+# Base template class that can be used by other templates.
+class AEBaseTemplate(ufeAeTemplate.Template):
```

Comment:
All the USD node templates use a base class (this one). It does all the work of creating the sections and adding the controls.